testing_and_setup/seaice/testing/tests/restartability_bgc.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
There is no need to create a whole new test script when it is almost identical to the original restartability script, simply add an option for bgc in the test suite, e.g.

			<test name="restartability_bgc">
			  <option name="bgc" value="True"/>
			</test>	

Then copy the functionality from the parallelism scrip for the multipleBlocks option e.g.

    bgcRestart = False
    if ("bgc" in options.keys() and options["bgc"] == "True"):
        bgcRestart = True

if (not bgcRestart):
    nmlChanges = {"seaice_model": {"config_start_time":"file"},
                  "restart":    {"config_do_restart":True}}
else:
    nmlChanges = {"seaice_model": {"config_start_time":"file"},
                  "restart":    {"config_do_restart":True,
                                 "config_do_restart_bgc":True, 
                                 "config_do_restart_hbrine":True}}